tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -452,17 +754,17 @@ public static ExplorationTestDescription GetExplorationTestDescription(Explorati
                 // Workaround for https://github.com/dotnet/runtime/issues/95653
                 EnvironmentVariables = new[] { ("DD_CLR_ENABLE_INLINING", "0") },
             },
-            //ExplorationTestName.ilspy => new ExplorationTestDescription()
-            //{
-            //    Name = ExplorationTestName.ilspy,
-            //    GitRepositoryUrl = "https://github.com/icsharpcode/ILSpy.git",
-            //    GitRepositoryTag = "v7.1",
-            //    IsGitSubmodulesRequired = true,
-            //    PathToUnitTestProject = "ICSharpCode.Decompiler.Tests",
-            //    IsTestedByVSTest = true,
-            //    TestsToIgnore = new[] { "UseMc", "_net45", "ImplicitConversions", "ExplicitConversions", "ICSharpCode_Decompiler", "NewtonsoftJson_pcl_debug", "NRefactory_CSharp", "Random_TestCase_1", "AsyncForeach", "AsyncStreams", "AsyncUsing", "CS9_ExtensionGetEnumerator", "IndexRangeTest", "InterfaceTests", "UsingVariables" },
-            //    SupportedFrameworks = new[] { TargetFramework.NET461 },
-            //},
+            ExplorationTestName.ilspy => new ExplorationTestDescription()
+            {
+                Name = ExplorationTestName.ilspy,
+                GitRepositoryUrl = "https://github.com/icsharpcode/ILSpy.git",
+                GitRepositoryTag = "v9.0-preview1",
+                IsGitSubmodulesRequired = true,
+                PathToUnitTestProject = "ICSharpCode.Decompiler.Tests",
+                IsTestedByVSTest = true,
+                TestsToIgnore = new[] { "UseMc", "_net45", "ImplicitConversions", "ExplicitConversions", "ICSharpCode_Decompiler", "NewtonsoftJson_pcl_debug", "NRefactory_CSharp", "Random_TestCase_1", "AsyncForeach", "AsyncStreams", "AsyncUsing", "CS9_ExtensionGetEnumerator", "IndexRangeTest", "InterfaceTests", "UsingVariables" },
+                /* SupportedFrameworks = new[] { TargetFramework.NET8_0_WIN }, */
```

Comment:
Is this accidental? If not we should just delete it

Reply:
Fixed in 6853dece4bb3cc91a684157e03ae7d3a18a3de0a